src/miner.erl

```diff
@@ -244,6 +245,21 @@ hbbft_status() ->
             end
     end.
 
+-spec hbbft_artifact() -> {ok, undefined} | {ok, binary()} | not_running | {error, timeout}.
+hbbft_artifact() ->
+   case gen_server:call(?MODULE, consensus_group, 60000) of
+        undefined -> not_running;
+        Pid ->
+            Ref = make_ref(),
+            ok = libp2p_group_relcast:handle_input(Pid, {get_artifact, Ref, self()}),
+            receive
+                {Ref, Result} ->
+                    Result
```

Comment:
should be weapped in ok tuple here, or in `handle_command({get_artifact`

Reply:
Fixed